relay/README.md

```diff
@@ -143,6 +143,7 @@ message CircuitRelay {
     HOP = 1;
     STOP = 2;
     STATUS = 3;
+    CAN_HOP = 4; // is peer a relay?
```

Comment:
When is this used? Won't the multistream handshake fail on the relay protocol eitherway?

Reply:
IIRC, this allows one to distinguish between a peer that simply accepts relayed connections and a peer that is willing to act as a relay.

Reply:
Here is the background of this change - https://github.com/libp2p/specs/issues/27. This was something that both @vyzo and myself thought is _important_ to have, that would otherwise complicate the code base.

Reply:
Sounds good. I understand where the need comes from (since we stopped using multicodecs to signal if nodes are hops or stops 🙏🏽).